docker/pipelines_runner.sh

```diff
@@ -73,13 +85,26 @@ function main {
     exit 1
   fi
 
+  # Build up the extra args is they are provided
+  if [[ ! -z "${subnetwork}" ]]; then
+    echo "Adding --subnetwork ${subnetwork} to optional_args"
+    pt_optional_args="${pt_optional_args} --subnetwork projects/${project}/regions/${region}/subnetworks/${subnetwork}"
+    df_optional_args="${df_optional_args} --subnetwork https://www.googleapis.com/compute/v1/projects/${project}/regions/${region}/subnetworks/${subnetwork}"
+  fi
+
+  if [[ ! -z "${use_public_ips}"  && "${use_public_ips}" == "true" ]]; then
```

Comment:
Shouldn't  this condition be `use_public_ips == "false"` ?

Reply:
Yes it should, my bad.

Good catch!

Reply:
Done